src/dmd/libelf.d

```diff
@@ -74,26 +74,26 @@ final class LibElf : Library
      * If the buffer is NULL, use module_name as the file name
      * and load the file.
      */
-    override void addObject(const(char)* module_name, const ubyte[] buffer)
+    override void addObject(const(char)[] module_name, const ubyte[] buffer)
     {
-        if (!module_name)
-            module_name = "";
         static if (LOG)
         {
-            printf("LibElf::addObject(%s)\n", module_name);
+            printf("LibElf::addObject(%s)\n",
+                   cast(int)module_name.length, module_name.ptr);
```

Comment:
```suggestion
                   cast(int) module_name.length, module_name.ptr);
```

Ditto below

Reply:
Hum, sorry, by the time I checked the D style this was merged. Actually forgot this was a rule. Will keep it in mind for future PRs.

Reply:
No big deal, just improves the readability IMHO